gradle/dependency-versions.gradle

```diff
@@ -23,13 +24,8 @@ ext {
     helixVersion = "1.1.0"
     jacksonVersion = "2.10.2"
     jaydioVersion = "0.1"
-    azureStorageBlobVersion = "12.15.0"
-    azureStorageBlobBatchVersion = "12.12.0"
-    azureCosmosDbAsyncVersion = "4.39.0"
     azureMsal4jVersion="1.7.1"
```

Comment:
Looks like we moved some azure libraries but not all, is that any reason for it?

Reply:
if u r referring to `com.microsoft.azure:msal4j`, its not part of azure-sdk.